src/NexusMods.App.UI/LeftMenu/Loadout/LoadoutLeftMenuViewModel.cs

```diff
@@ -113,6 +118,23 @@ public class LoadoutLeftMenuViewModel : AViewModel<ILoadoutLeftMenuViewModel>, I
                     diagnosticItem.Name = $"{Language.LoadoutLeftMenuViewModel_LoadoutLeftMenuViewModel_Diagnostics} ({totalCount})";
                 })
                 .DisposeWith(disposable);
+            
+            loadoutRegistry.LoadoutRootChanges
+                .OnUI()
+                .OnItemRemoved(loadoutId =>
+                {
+                    if (workspaceController.ActiveWorkspace?.Context is LoadoutContext activeLoadoutContext &&
+                        activeLoadoutContext.LoadoutId == loadoutId)
+                    {
+                        workspaceController.ChangeOrCreateWorkspaceByContext<HomeContext>(() => new PageData
+                        {
+                            FactoryId = MyGamesPageFactory.StaticId,
+                            Context = new MyGamesPageContext(),
+                        });
+                    }
+                }, false)
+                .SubscribeWithErrorLogging()
+                .DisposeWith(disposable);
```

Comment:
I think this should be on the spine rather than on the left menu

Reply:
Good point.
I had it in left menu due to how an earlier iteration works, but current code is more fitting for spine.